packages/terra-toggler/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "terra-toggler",
+  "main": "lib/Toggler.js",
+  "private": true,
+  "version": "0.0.0",
+  "description": "Toggler component that transitions content in and out.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-toggler",
+    "Toggler",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "terra-toolkit": "^0.x"
+  },
+  "peerDependencies": {
+    "react": "^15.5.4",
+    "react-dom": "^15.5.4",
+    "terra-base": "^0.x",
+    "terra-mixins": "^1.0.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "react-animate-height": "^0.9.5",
```

Comment:
I'd prefer to use this dependency for animating height. Using CSS we're limited to using something like animating to `max-height: 1000px`. While this works, if content is over 1000px for whatever reason, the CSS approach breaks down. With react-transition-group, I believe we'd still be limited to CSS max-height animation. 
This post influenced my decision to pull in this dependency: https://stanko.github.io/react-animate-height-slide-up-down/